Public/Src/Cache/ContentStore/Library/Utils/IpcUtilities.cs

```diff
@@ -183,10 +200,13 @@ private static class EventWaitHandleAccessRules
         {
             public static EventWaitHandleAccessRule PublicSynchronizeAccessRule()
             {
-                var currentUser = UserUtilities.CurrentUserName();
+                // In CoreCLR, EventWaitHandle.TryOpenExisting() does not support setting access rights explicitly.
+                // Instead, a handle is always opened with EventWaitHandleRights.Synchronize | EventWaitHandleRights.Modify

```

Comment:
I know this makes it 'work' but it does open up a security vulnerability... If BuildXL server process runs, now any process of any user on the machine has the ability to unblock the waithandle and cause issues for the user building.

Reply:
True, any process can call `Set()` on it with this change.  We can revert this change and just go with the hacky `EventWaitHandleTryOpenExisting` (that works too).

Reply:
reverted